utils/wtmcu/files/wtmcud.init

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh /etc/rc.common
+
+START=10
```

Comment:
Why does it need to start this early?
Preferably set 99 instead as it's not system critical

Reply:
It is kind of system critical. MCU is in charge of controlling fan, temperature, power for peripherals, LEDs, WOL, APR and battery-backed clock. I'd like it to be available as early as possible as it provides ubus service to control all of this.

Reply:
Sure, but to my knowledge ubus depends on rpcd which is started later on?
https://github.com/openwrt/openwrt/blob/master/package/system/rpcd/files/rpcd.init#L3

Reply:
Well, yes and no. If your service is a plugin for rpcd (either one-off or library) you have to wait for rpcd. But wtmcud is a generic ubus service, which connects directly and does not require rpcd. Ubus is a part of procd and is already available just before the init stage.